tests/wpt/mozilla/tests/appearance/input-text-definite-width.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Appearance of Input type=text With a Definite Width</title>
+    <link rel="match" href="input-text-definite-width-ref.html">
+    <link rel="help" href="https://github.com/servo/servo/pull/37065">
+</head>
+<body>
+Display of input type=text should match the display generated by the CSS reference.
+<div>
+    <input type="text" value="Foo" style="font-size: 1em !important; width: 100px;"></input>
```

Comment:
```suggestion
    <input type="text" value="Foo" style="width: 100px;"></input>
```
Setting `font-size: 1em` should be effectively a noop. Ditto to the ref file.

Reply:
I add these because the layout has some small inconsistencies in terms of placing the "input" box inside its parent. 

This, however, is not replicable in Firefox and Chrome.